src/service/resources-impl.js

```diff
@@ -1495,8 +1476,7 @@ export class Resource {
     /** @const {!AmpElement|undefined|null} */
     this.owner_ = undefined;
 
-    /** @const {number} */
-    this.priority_ = getElementPriority(element.tagName);
+    this.priority_ = element.getElementPriority();
```

Comment:
If I add the assertion in the `ElementProto` in `custom-element.js`, then I don't need to override `getPriority()` in `StubElement` with an exception. Is that correct? Since `getPriority()` in `StubElement` has no chance to be called anywhere.